docs/outputs.rst

```diff
@@ -142,10 +142,47 @@ sampled to those subject spaces.
 Confounds
 ---------
 
-See implementation on :mod:`~fmriprep.workflows.bold.confounds.init_bold_confs_wf`.
-
-
-For each :abbr:`BOLD (blood-oxygen level dependent)` run processed with fMRIPrep, a
+The :abbr:`BOLD (blood-oxygen level dependent)` signal measured with fMRI is a mixture of fluctuations
+of both neuronal and non-neuronal origin.
+Neuronal signals are measured indirectly as changes in the local concentration of oxygenated hemoglobin.
+Non-neuronal fluctuations in fMRI data may appear as a result of head motion, scanner noise,
+or physiological fluctuations (related to cardiac or respiratory effects).
+For a detailed review of the possible sources of noise in the BOLD signal, refer to [Greve2013]_.
+
+*Confounds* (or nuisance regressors) are variables representing fluctuations with a potential non-neuronal origin.
+Such non-neuronal fluctuations may drive spurious results in fMRI data analysis,
+including standard activation :abbr:`GLM (General Linear Model)` and functional connectivity analyses.
+It is possible to minimize confounding effects of non-neuronal signals by including them as nuisance regressors
+in the :abbr:`GLM (General Linear Model)` design matrix or regressing them out from
+the fMRI data - a procedure known as *denoising*.
+There is currently no consensus on an optimal denoising strategy in the fMRI community.
+Rather, different strategies have been proposed, which achieve different compromises between
+how much of the non-neuronal fluctuations are effectively removed, and how much of neuronal fluctuations
+are damaged in the process.
+The *fMRIPrep* pipeline generates a large array of possible confounds.
```

Comment:
```suggestion
The *fMRIPrep* pipeline generates a large array of possible confounds.
Note, *fMRIPrep* does not perform any denoising itself and it is up to the user to perform this step.
```
My thinking is this should be explictly stated. Not sure if this is the best formulation. But a suggestion